### ISSUE TITLE
"Réorienter une blessure à la tête"

### ISSUE THREAD

**Issue by brunobord** [lecrapal/lecrapal.github.io#6]
Le terme me paraît confus… mais je n'ai pas de meilleure traduction. Cette _issue github_ pourrait ouvrir une discussion pour éventuellement trouver plus clair ?

en VO:

> Reorienting Head Wound

Moi je le comprends comme "un coup à la tête qui réoriente / secoue / bouleverse", ce qui explique qu'on ouvre la possibilité d'augmenter une caractéristique.

Mais alors, comment transformer cette idée en expression compréhensible ?… mystère.

**Comment by lecrapal** [comment]
Oui, exactement, bien vu. Je t'avoue que quand j'ai traduit ce terme j'étais pas très fier du résultat.
J'ai pensé à "Blessure à la tête désorientante" ou "Traumatisme crânien déstabilisant", mais pareil, ça sonne bizarrement, peut être trop verbeux.
Ou alors plus simple : "Choc crânien", on perd la notion écrite du fait que ce soit désorientant, mais bon, ça peut l'induire.

**Comment by brunobord** [comment]
"tempête dans le crâne" ?

**Comment by lecrapal** [comment]
"Rétablissement suite à un coup à la tête" ?
C'est un peu long mais bon...

(oui j'y reviens 1 an après, désolé de la longue absence)

**Comment by brunobord** [comment]
mmmm… "Remue-méninges" ?

(en fait c'est tordu, parce que ça implique que le choc reçu à la tête va _améliorer_ quelque chose au lieu de réduire les capacités du PJ)
